products/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+
+class Category(models.Model):
+    category = models.CharField(max_length=20)
```

Comment:
우선 대부분의 모델이 위와 같이 작성되어 있는데 name으로 컬럼명을 바꿔줄 필요가 있어보입니다.
해당되는 모든 모델 수정해주세요.
```python
category = Category.objects.get(id=1)
(X) category.category
(O) category.name
```
```suggestion
class Category(models.Model):
    name = models.CharField(max_length=20)
```

Reply:
리뷰 반영하여 수정했습니다. 검토부탁드리겠습니다~!